akka-stream-tests/src/test/scala/akka/stream/impl/fusing/InterpreterSpec.scala

```diff
@@ -79,9 +79,9 @@ class InterpreterSpec extends AkkaSpec with GraphInterpreterSpecKit {
       lastEvents() should be(Set(OnComplete))
     }
 
-    "implement one-to-many many-to-one chain correctly" in new OneBoundedSetup[Int](Seq(
-      Doubler(),
-      Filter((x: Int) ⇒ x != 0, stoppingDecider))) {
+    "implement one-to-many many-to-one chain correctly" in new OneBoundedSetup[Int](
+      Doubler().toGS,
```

Comment:
Would be even better to convert those to GraphStages directly so we can move on with dropping PushPullStage :)